ci/firebase_testlab.py

```diff
@@ -66,7 +68,7 @@ def check_logcat(results_dir):
   logcat = subprocess.check_output([
       'gsutil', 'cat',
       '%s/%s/*/logcat' % (BUCKET, results_dir)
-  ])
+  ]).decode(ENCODING)
```

Comment:
Let's try with:

```
logcat = subprocess.check_output([ ....
logcat = logcat if isinstance(logcat, str) else logcat.decode(ENCODING)
```

Reply:
Okay no problem but I thought the exercise was that we were no longer targeting python 2?

Reply:
Okay gotcha. I though we going full on to python 3. Let me make the changes.